packages/material-ui/src/MobileStepper/MobileStepper.test.js

```diff
@@ -1,22 +1,16 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import {
-  getClasses,
-  createMount,
-  createClientRender,
-  describeConformance,
-  screen,
-} from 'test/utils';
+import { createMount, createClientRender, describeConformanceV5, screen } from 'test/utils';
 import KeyboardArrowLeft from '../internal/svg-icons/KeyboardArrowLeft';
 import KeyboardArrowRight from '../internal/svg-icons/KeyboardArrowRight';
 import Paper, { paperClasses } from '../Paper';
 import Button from '../Button/Button';
 import MobileStepper from './MobileStepper';
+import classes from './mobileStepperClasses';
```

Comment:
I would propose we use the opportunity to import from the public API

```suggestion
import Paper, { paperClasses } from '@material-ui/core/Paper';
import Button from '@material-ui/core/Button';
import MobileStepper, { mobileStepperClasses as classes } from '@material-ui/core/MobileStepper';
```